modules/tests/jvm/src/main/scala/gsp/math/geom/jts/demo/JtsDemo.scala

```diff
@@ -4,45 +4,49 @@
 package lucuma.core.math.geom.jts
 package demo
 
-import lucuma.core.math.{Angle, Offset}
+import lucuma.core.enum.PortDisposition
+import lucuma.core.enum.GmosSouthFpu
+import lucuma.core.enum.GmosNorthFpu
+import lucuma.core.math.{ Angle, Offset }
+import lucuma.core.math.geom.GmosOiwfsProbeArm
+import lucuma.core.math.geom.GmosScienceAreaGeometry
 import lucuma.core.math.geom.ShapeExpression
 import lucuma.core.math.geom.jts.jvm.syntax.awt._
 import lucuma.core.math.geom.jts.interpreter._
 import lucuma.core.math.geom.syntax.shapeexpression._
 import lucuma.core.math.syntax.int._
 
-import java.awt.{List => _, _}
+import java.awt.{ List => _, _ }
 import java.awt.event._
 
 import scala.jdk.CollectionConverters._
 
 /**
- * Throwaway demo code to visualize a shape created using `ShapeExpression`s.
- */
+  * Throwaway demo code to visualize a shape created using `ShapeExpression`s.
+  */
 object JtsDemo extends Frame("JTS Demo") {
 
-  val posAngle: Angle         =
+  val posAngle: Angle =
     145.deg
 
   val guideStarOffset: Offset =
     Offset(170543999.µas.p, -24177003.µas.q)
 
-  val offsetPos: Offset       =
-    Offset(60.arcsec.p, 60.arcsec.q)
+  val offsetPos: Offset =
+    Offset(-60.arcsec.p, 60.arcsec.q)
 
-  // TODO: should come from the FPUnit enum in core
-  val ifuOffset: Offset       =
-    Offset.Zero
+  val fpu: Option[Either[GmosNorthFpu, GmosSouthFpu]] =
+    Some(Right(GmosSouthFpu.LongSlit_5_00))
 
-  // TODO: when we move this core there should be an enum
-  val sideLooking: Boolean    =
-    true
+  val port: PortDisposition =
+    PortDisposition.Side
 
   // Shape to display
   val shapes: List[ShapeExpression] =
     List(
-      GmosOiwfsProbeArm.shapeAt(posAngle, guideStarOffset, offsetPos, ifuOffset, sideLooking),
-      GmosOiwfsProbeArm.patrolFieldAt(posAngle, offsetPos, ifuOffset, sideLooking),
+      GmosOiwfsProbeArm.shapeAt(posAngle, guideStarOffset, offsetPos, fpu, port),
+      GmosOiwfsProbeArm.patrolFieldAt(posAngle, offsetPos, fpu, port),
+      GmosScienceAreaGeometry.imaging ⟲ posAngle,
       GmosScienceAreaGeometry.imaging ↗ offsetPos ⟲ posAngle
```

Comment:
This all looks good, but I am curious whether all the changes in the old `gsp-core` got picked up.  It evolved a bit in `gsp-core` after I added it there.  It _really doesn't matter_ for `JtsDemo` but it'd be good to know all the substantive changes to the rest are there.

Here for example in line 50 it is

```scala
GmosScienceAreaGeometry.shapeAt(posAngle, offsetPos, fpu)
```

in `gsp-core`.

Reply:
`GmosOiwfsProbeArm` and `GmosScienceAreaGeometry` were copied from `gsp-core`. The only changes were the imports. Are there other classes from `gsp-core` that I'm missing?